### PR TITLE
For 11

### PR DESCRIPTION
**Pull request by afolasope**

### CODE REVIEW

File: src/surveys/components/TextResponse.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  return (
+    <div className={'max-w-[440px]'}>
+      <Header
+        title={question.label}
+        description={question.description}
+        color={theme?.question}
+      />
+      <div>
+        <form onSubmit={onSubmitHandler}>
+          <textarea
```
Comment (jdamilola) [inline src/surveys/components/TextResponse.tsx]: @afolasope Can you add `singleLine` flag to `TextSettings`, I totally missed this. So you render an `input` field if it's `true` but a `textarea` if it's `false`.

Comment (afolasope) [reply src/surveys/components/TextResponse.tsx]: What is the default value?

Comment (jdamilola) [reply src/surveys/components/TextResponse.tsx]: @afolasope The value will always be `false` until it's set to `true`

Comment (afolasope) [reply src/surveys/components/TextResponse.tsx]: I have done that

File: src/surveys/components/TextResponse.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        <form onSubmit={onSubmitHandler}>
+          {(question.settings as TextSettings).singleLine === true ? (
+            <input
+              type="text"
+              className={
+                'w-full mt-3 border border-gray-300 rounded-xl p-3 bg-formily-grey focus:outline-none focus:ring-2 focus:border-transparent'
+              }
+              style={{
+                color: theme?.question,
+                backgroundColor: theme?.answer,
```
Comment (jdamilola) [inline src/surveys/components/TextResponse.tsx]: @afolasope The `color` should be `theme?.answer` and the `backgroundColor` should also be `theme?.answer` but with 10% opacity.

File: src/surveys/components/TextResponse.tsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+                onChange={(
+                  e: h.JSX.TargetedEvent<HTMLTextAreaElement, Event>
+                ) => {
+                  setAnswer(e.currentTarget.value);
+                }}
+                className={
+                  'w-full mt-3 resize-none border border-gray-300 rounded-xl p-4 bg-formily-grey focus:outline-none focus:ring-2 focus:border-transparent'
+                }
+                style={{
+                  color: theme?.question,
```
Comment (jdamilola) [inline src/surveys/components/TextResponse.tsx]: @afolasope The `color` should be `theme?.answer` and the `backgroundColor` should also be `theme?.answer` but with 10% opacity.

**Review by jdamilola — APPROVED** [review]

**Pull request merged**